src/Controller/AppController.php

```diff
@@ -182,7 +182,11 @@ protected function prepareRequest($type): array
                 $id = $data['id'];
 
                 foreach ($relationData as $method => $ids) {
-                    $relatedIds = json_decode($ids, true);
+                    $relatedIds = is_string($ids) ? json_decode($ids, true) : array_map(
```

Comment:
In order to ease debugging we avoid ternary `?` operator usually... 

```php
     if (is_string($ds)) {
           $relatedIds = is_string($ids);
     } else { 
           $relatedIds = array_map(...);
     }
```

but you can keep it like this